test/Test/Main.purs

```diff
@@ -0,0 +1,68 @@
+module Test.Main where
+
+import Prelude
+
+import Test.Assert (ASSERT, assert)
+import Partial.Unsafe (unsafeCrashWith)
+import Control.Monad.Eff (Eff)
+import Control.Monad.Eff.Console (CONSOLE, log)
+import Control.Monad.Gen (class MonadGen, frequency)
+import Data.NonEmpty ((:|), NonEmpty(..))
+import Data.Traversable (sequence)
+import Data.Array (replicate, group', length)
+import Data.Tuple (Tuple(..))
+import Data.Newtype (unwrap)
+import Math (remainder)
+import Control.Monad.State (State, class MonadState, get, put, evalStateT)
+
+type TestEffects = (console :: CONSOLE, assert :: ASSERT)
+type Tests = Eff TestEffects Unit
+
+newtype TestGen a = TestGen (State Number a)
```

Comment:
Can we add a comment to this pointing out it's only suitable for testing `frequency`? :smile:

Reply:
✅